### PR TITLE
[BT-656]: Fix ModalStack so that it doesn't crop overflowing content.

### PR DESCRIPTION
**Pull request by cjcenizal**
Update example to demo this.

### CODE REVIEW

File: src/guide/views/dropdown/DropdownExample.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default class DropdownExample extends Component {
     }, {
       name: 'Berry',
     }, {
-      name: 'Chippewa',
+      name: 'Corn',
```
Comment (cjcenizal) [inline src/guide/views/dropdown/DropdownExample.jsx]: Chippewa is the name of a Native American tribe, but everyone else is something edible... one of these things was not like the others. 😮

Comment (ahollesen) [inline src/guide/views/dropdown/DropdownExample.jsx]: Technically daffodils are edible, but more a flower than food right?! ;)

**Comment by ahollesen** [comment]
Looking good 👍

**Comment by jenslonkowski** [comment]
Great! 👍

**Pull request merged**